README.md

```diff
@@ -1 +1,55 @@
-# scm-router
\ No newline at end of file
+# SCM Router
+[![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][status-image]][status-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
+
+> A generic scm plugin that routes builds to a specified scm
+
+A scm is interface for interacting with a source control management system.
```

Comment:
"An SCM is a source control management system, like Github, Bitbucket, Gitlab, etc. This SCM Router module will allow multiple SCMs to be used within a single Screwdriver cluster."